salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
Then what is the effect of `stateful: True`? The real chain of actions in not `write changes -> daemon-reload -> restart docker` but `write changes -> daemon-reload && restart docker`, so I think nobody cares about the changes produced by `daemon-reload`, so we wouldn't have to set `stateful: True`.